charts/csi-unity/values.yaml

```diff
@@ -31,7 +31,7 @@ images:
 # LogLevel is used to set the logging level of the driver.
 # Allowed values: "error", "warn"/"warning", "info", "debug"
 # Default value: "info"
-logLevel: "info"
+logLevel: "debug"
```

Comment:
Is there a specific reason or requirement for setting the default value to 'debug' here? While I understand that this PR aims to ensure consistency across all drivers, wouldn't this lead to unnecessary logging unless explicitly needed by the customer? Personally, as a developer, I’d prefer having 'debug' as the default, as it can be very useful for troubleshooting issues😊but not for all users.

If consistency is the goal, perhaps we could consider setting 'info' as the default value across all drivers instead?